src/components/ActiveVideo/Confirm/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { h, FunctionComponent } from 'preact'
+import type { WithLocalisedProps, WithTrackingProps } from '~types/hocs'
+import { useEffect, useState } from 'preact/hooks'
+import RecordingComplete from '../RecordingComplete'
+import Uploading from '../Uploading'
+import ConnectionError from '../ConnectionError'
+import { StepComponentProps } from '~types/routers'
+import { localised } from '~locales'
+
+type Props = StepComponentProps & WithLocalisedProps & WithTrackingProps
+
+enum ConfirmationStep {
+  RECORDING_COMPLETE,
+  UPLOADING,
+  UPLOAD_ERROR,
+}
+
+const Confirm: FunctionComponent<Props> = (props: Props) => {
+  const { previousStep } = props
+
+  const [confirmationStep, setConfirmationStep] = useState<ConfirmationStep>(
+    ConfirmationStep.RECORDING_COMPLETE
+  )
+
+  useEffect(() => {
+    setConfirmationStep(ConfirmationStep.RECORDING_COMPLETE)
+  }, [])
+
+  console.log(confirmationStep)
```

Comment:
Should this be here?

Reply:
Oops! Not at all, thanks for spotting this :bow: